environments/env_generator/examples/multi_customer.json

```diff
@@ -0,0 +1,189 @@
+            "x": 6,
+            "y": 5
+        },
+        {
+            "name": "table",
+            "x": 1,
+            "y": 7
+        },
+        {
+            "name": "table",
+            "x": 2,
+            "y": 7
+        },
+        {
+            "name": "board",
+            "x": 3,
+            "y": 7
+        },
+        {
+            "name": "table",
+            "x": 4,
+            "y": 7
+        },
+        {
+            "name": "table",
+            "x": 5,
+            "y": 7
+        },
+        {
+            "name": "table",
+            "x": 6,
+            "y": 7
+        },
+        {
+            "name": "customertable",
+            "x": 1,
+            "y": 1
+        },
+        {
+            "name": "customerspawn",
+            "x": 7,
+            "y": 1
+        }
+    ],
+    "items": [
+        {
+            "name": "patty",
+            "x": 2,
+            "y": 5,
+            "stack-level": 0,
+            "predicates": ["iscookable"]
+        },
+        {
+            "name": "patty",
+            "x": 4,
+            "y": 5,
+            "stack-level": 0,
+            "predicates": ["iscookable"]
+        },
+        {
+            "name": "lettuce",
+            "x": 2,
+            "y": 7,
+            "stack-level": 0,
+            "predicates": ["iscuttable"]
+        },
+        {
+            "name": "lettuce",
+            "x": 4,
+            "y": 7,
+            "stack-level": 0,
+            "predicates": ["iscuttable"]
+        },
+        {
+            "name": "bottombun",
+            "x": 5,
+            "y": 5,
+            "stack-level": 0
+        },
+        {
+            "name": "topbun",
+            "x": 5,
+            "y": 7,
+            "stack-level": 0
+        },
+        {
+            "name": "bottombun",
+            "x": 6,
+            "y": 5,
+            "stack-level": 0
+        },
+        {
+            "name": "topbun",
+            "x": 6,
+            "y": 7,
+            "stack-level": 0
+        }
+    ],
+    "players": [
+        {
+            "name": "robot",
+            "x": 1,
+            "y": 4,
+            "direction": [0, 1]
+        }
+    ],
+    "meals": [],
+    "containers": [
+        {
+            "name": "bowl",
+            "x": 1,
+            "y": 5
+        }
+    ],
+    "customers": [
+        {
+            "name": "customer",
+            "x": 7,
+            "y": 0,
+            "direction": [-1, 0],
+            "order": "lettuceburger",
+            "time_to_serve": 45000,
+            "enter_time": 5000
+        },
+        {
+            "name": "customer",
+            "x": 7,
+            "y": 0,
+            "direction": [-1, 0],
+            "order": "lettuceburger",
+            "time_to_serve": 1000000,
+            "enter_time": 10000
+        }
+    ],
+    "goal_description": "Serve the customer",
+    "goal": [
+        {
+            "predicate": "iscookable",
+            "args": ["lettuce"],
```

Comment:
I also just played through these and realized the goal is unrealizable. What is the plan for checking that these environments are done?

Reply:
My bad, did this to make it easier to debug. Edited the goals accordingly to close when all customers have left the game